js/src/elements/star-rating.ts

```diff
@@ -314,6 +316,9 @@ export class StarRatingElement extends HTMLElement {
 				this.updateContent('value', target.value, target);
 				this.dispatchEvent(new Event('change', { bubbles: true }));
 				break;
+			case 'focus':
+				this.shadowRoot!.querySelectorAll('input')[0].focus();
```

Comment:
When I click on the fourth star, the first star gets a focus-visible outline, this is desireable?

Reply:
No that was an unintended side effect, should be fixed now.